socrates/lib/site/index.js

```diff
@@ -17,110 +17,124 @@ const sponsorpairs = require('./sponsorpairs');
 const app = misc.expressAppIn(__dirname);
 
 app.get('/', (req, res, next) => {
-  eventstoreService.getRegistrationReadModel(socratesConstants.currentUrl, (err, registrationReadModel) => {
-    if (err || !registrationReadModel) { return next(err); }
-    eventstoreService.getRoomsReadModel(socratesConstants.currentUrl, (err2, roomsReadModel) => {
-      if (err2 || !roomsReadModel) { return next(err2); }
-      const memberId = res.locals.accessrights.memberId();
-      const registration = {
-        alreadyRegistered: registrationReadModel.isAlreadyRegistered(memberId),
-        alreadyOnWaitinglist: registrationReadModel.isAlreadyOnWaitinglist(memberId),
-        selectedOptions: registrationReadModel.selectedOptionsFor(memberId),
-        roommate: roomsReadModel.roommateFor('bed_in_double', memberId) || roomsReadModel.roommateFor('bed_in_junior', memberId)
-      };
-
-      res.render('index', {sponsors: sponsorpairs(), registration});
-    });
+  return eventstoreService.getRegistrationReadModel(socratesConstants.currentUrl, (err, registrationReadModel) => {
+    if (err || !registrationReadModel) {
+      return next(err);
+    } else {
+      return eventstoreService.getRoomsReadModel(socratesConstants.currentUrl, (err2, roomsReadModel) => {
+        if (err2 || !roomsReadModel) {
+          return next(err2);
+        } else {
+          const memberId = res.locals.accessrights.memberId();
+          const registration = {
+            alreadyRegistered: registrationReadModel.isAlreadyRegistered(memberId),
+            alreadyOnWaitinglist: registrationReadModel.isAlreadyOnWaitinglist(memberId),
+            selectedOptions: registrationReadModel.selectedOptionsFor(memberId),
+            roommate: roomsReadModel.roommateFor('bed_in_double', memberId) || roomsReadModel.roommateFor('bed_in_junior', memberId)
+          };
+
+          return res.render('index', {sponsors: sponsorpairs(), registration});
+        }
+      });
+    }
   });
 });
 
 app.get('/goodbye.html', (req, res) => {
   if (req.user && req.user.member) {
     return res.redirect('/');
+  } else {
+    return res.render('goodbye');
   }
-  res.render('goodbye');
 });
 
 app.get('/robots.txt', (req, res, next) => {
-  fs.readFile(path.join(__dirname, 'views', 'robots.txt'), 'utf8', (err, data) => {
-    if (err) { return next(err); }
-    res.send(data);
+  return fs.readFile(path.join(__dirname, 'views', 'robots.txt'), 'utf8', (err, data) => {
+    if (!err) {
+      return res.send(data);
+    } else {
+      return next(err);
+    }
   });
 });
 
 app.get('/impressum.html', (req, res) => {
-  res.render('impressum');
+  return res.render('impressum');
 });
 
 app.get('/schedule.html', (req, res) => {
-  res.render('schedule', {currentYear: socratesConstants.currentYear});
+  return res.render('schedule', {currentYear: socratesConstants.currentYear});
```

Comment:
Das macht es in meinen Augen nicht besser. Besser wäre hier, den Block um den function body zu entfernen und das ganze zum Einzeiler zu machen:
`(req, res) => res.render...`